nautobot/core/templates/generic/object_create.html

```diff
@@ -9,8 +10,8 @@
             {{ field }}
         {% endfor %}
         <div class="row">
-            <div class="col-md-6 col-md-offset-3">
-                <h3>
+            <div class="col-md-10 col-md-offset-1 col-lg-8 col-lg-offset-2">
```

Comment:
Probably should revert the changes here and below and plan to address them more holistically as part of #5045 and/or #5062.